### PR TITLE
Viewing organization profile

### PR DESCRIPTION
**Pull request by BadryAtef**
organization profile almost done except events and reviews because i didn't find routes or views to display them

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -35,6 +35,14 @@ public function subscribe($organization_id){
         if(!$this->subscribedOrganizations()->find($organization_id))
             $this->subscribedOrganizations()->attach($organization_id);
     }
+    /**
+     * to check if the user already subscribed an organization
+     */
+    public function is_subscribed($id){
+        if(!$this->subscribedOrganizations()->find($id))
```
Comment (AhmadElsagheer) [inline app/User.php:42]: isSubscribed

Comment (BadryAtef) [inline app/User.php:42]: da 34an convention mo3ayn wla leha 8rd tany

Comment (AhmadElsagheer) [inline app/User.php:42]: convention

**Comment by AhmadElsagheer** [comment]
Good Job :+1: :+1:

**Pull request merged**